website/get-involved/index.html

```diff
@@ -0,0 +1,172 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "_includes/base-resp.html" %}
+
+{% block page_title %}{{ _('Get Involved') }}{% endblock %}
+
+{% block page_desc %}
+{% if LANG.startswith('en-') %}
+Everything you need to know in order to get involved with Thunderbird.
+{% else %}
+{{ _('Everything you need to know in order to get involved with Thunderbird.') }}
+{% endif %}
+{% endblock %}
+
+{% block body_id %}get-involved{% endblock %}
+
+{% block page_css %}
+<link href="/media/css/thunderbird-features.css" rel="stylesheet" type="text/css" />
+{% endblock %}
+
+{% block content %}
+<header id="main-feature">
+    <h1>{{ self.page_title() }}</h1>
+    <h2>{{ self.page_desc() }}</h2>
+</header>
+
+<nav role="directory">
+    <ul class="feature-links">
+        <li><a href="#development">{{ _('Development') }}</a></li>
+        <li><a href="#documentation">{{ _('Documentation') }}</a></li>
+        <li><a href="#communication">{{ _('Communication') }}</a></li>
+        <li><a href="#infrastructure">{{ _('Infrastructure') }}</a></li>
+        <li><a href="#design">{{ _('Design') }}</a></li>
+        <li><a href="#translation">{{ _('Translation') }}</a></li>
+        <li><a href="#qualityassurance">{{ _('Quality Assurance') }}</a></li>
+        <li><a href="#marketing">{{ _('Marketing') }}</a></li>
+    </ul>
+</nav>
+
+<main role="main">
+    <section id="development">
+        <h2>{{ _('Development') }}</h2>
+        <div class="row">
+            <div class="col full">
+                <section>
+                    <h3>{{ _('Code') }}</h3>
+                    <p>{{ _('Code is located on Mozilla\'s Mercurial server.') }}</p>
+                    <ul>
+                        <li><a href="https://hg.mozilla.org/comm-central/">{{ _('comm-central') }}</a> {{ _('is the main Thunderbird repository.') }}</li>
+                    </ul>
+                </section>
+                <section>
+                    <h3>{{ _('Getting Started') }}</h3>
+                    <p>{{ _('Read the') }} <a href="https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Introduction">{{ _('Developer Guide') }}</a> {{ _('on MDN. That will give you a general overview of the ecosystem Thunderbird is part of.') }}</p>
```

Comment:
In general, one sentence should be one gettext call, beccause localizers can't effectively translate a bunch of isolated sentence fragments. <a> tags can be included in the localized text, and should be so, localizers have some context for what the link is. See: https://github.com/thundernest/thunderbird-website/blob/master/website/features/index.html#L206

If the whole sentence is a link, like the brief phrases on the front page describing links underneath the download button, it's OK to exclude the <a> tag like: https://github.com/thundernest/thunderbird-website/blob/master/website/_includes/download-button.html#L57

Reply:
Thanks, I was wondering about that. It did seem kinda silly to do it the way I did. :)